ihmc-perception/src/main/java/us/ihmc/perception/imageMessage/ImageMessageDecoder.java

```diff
@@ -31,6 +39,30 @@ public ImageMessageDecoder()
       }
    }
 
+   public RawImage decodeMessage(ImageMessage messageToDecode)
```

Comment:
Not sure why I didn't make a method that returns a `RawImage` in the first place. Only dilemma I have is whether to use a CPU or GPU mat. I went with CPU since this class is used on the UI side. Should I add a method that uses GPU mat too?

Reply:
I don't think GPU is needed

Reply:
I guess in the method name you should put CPU